WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -957,6 +959,10 @@ import Foundation
             return "domains_dashboard_viewed"
         case .domainsDashboardAddDomainTapped:
             return "domains_dashboard_add_domain_tapped"
+        case .domainsDashboardGetDomainTapped:
+            return "domains_dashboard_get_domain_tapped"
```

Comment:
Wasn't this `domains_dashboard_get_domain_tapped` event already there before!

Reply:
No, it wasn't. I checked Android events and made them match.